frame/authority-discovery/src/lib.rs

```diff
@@ -206,12 +214,21 @@ mod tests {
 			.map(|i| AuthorityPair::from_seed_slice(vec![i; 32].as_ref()).unwrap().public())
 			.map(AuthorityId::from)
 			.collect();
-
 		// Needed for `pallet_session::OneSessionHandler::on_new_session`.
-		let second_authorities_and_account_ids: Vec<(&AuthorityId, AuthorityId)> = second_authorities.clone()
+		let second_authorities_and_account_ids = second_authorities.clone()
```

Comment:
Should technically have the same set as the first one, but anyhow fine 

https://github.com/paritytech/substrate/blob/549050b7f1740c90855e777daf3f9700750ad7ff/frame/session/src/lib.rs#L225-L229